apps/mobile/src/queries/balance/bitcoin-balance.query.ts

```diff
@@ -54,7 +56,7 @@ type TotalBalanceCombineFn = UseQueryResult<
 
 export function useTotalBitcoinBalanceOfDescriptors(descriptors: string[]) {
   const queries = useCreateBitcoinAccountUtxoQueryOptions(descriptors);
-  const calcUsdAmount = useCalculateBitcoinFiatValue();
+  const { data: btcMarketData } = useBtcMarketDataQuery();
```

Comment:
Nice work integrating this 👍 I can use it in the STX balance too. That way the service becomes integrated and used already